questionpy_sdk/webserver/app.py

```diff
@@ -84,6 +87,7 @@ async def start_server(self) -> None:
         self._runner = web.AppRunner(self._web_app)
         await self._runner.setup()
         await web.TCPSite(self._runner, self._host, self._port).start()
+        self._print_urls()
```

Comment:
Was spricht dagegen, einfach nur `f"http://{self._host}:{self._port}"` auszugeben?

Reply:
Das würde nicht alle Adressen auflisten, auf dem der Webserver horcht:

```
$ questionpy-sdk run examples/minimal
INFO:questionpy-sdk:web-server:Webserver started: http://[::1]:8080 http://127.0.0.1:8080
```

versus

```
$ questionpy-sdk run examples/minimal
INFO:questionpy-sdk:web-server:Webserver started: http://localhost:8080
```

Reply:
Schon klar, aber ist das nötig? Bei den anderen Tools die ich kenne, die mir ihre URL in der Konsole ausgeben, wird mir immer nur eine angezeigt.

Reply:
> Schon klar, aber ist das nötig? Bei den anderen Tools die ich kenne, die mir ihre URL in der Konsole ausgeben, wird mir immer nur eine angezeigt.

Ich hatte es auch erst genauso implementiert, wie du es geschrieben hattest. Hab's dann noch mal geändert, da ich dachte, es sei etwas eleganter die Info direkt aus dem Server herauszunehmen.

Allerdings macht das den Code natürlich länger. Für Menschen die IPv6 nutzen, macht es Sinn. Für alle anderen nicht, denke ich...

Bin da emotionslos. Ändere das aber gerne um. Ist ja quasi dein Feature :)